src/test/java/picard/analysis/CollectWgsMetricsTestUtils.java

```diff
@@ -7,39 +7,30 @@
 import java.io.File;
```

Comment:
this file doesn't seem to be licensed...I know it's not your fault....please fix it.